src/data.jl

```diff
@@ -1,9 +1,32 @@
 export
-    get_burgers_data
+    UnitGaussianNormalizer,
+    Encode,
+    Decode,
+    get_burgers_data,
+    get_darcy_flow_data
 
-function register_datasets()
+struct UnitGaussianNormalizer{T}
+    mean::Array{T}
+    std::Array{T}
+    ϵ::T
+end
+
+function UnitGaussianNormalizer(𝐱; ϵ=1f-5)
+    dims = 1:length(size(𝐱))-1
+
+    return UnitGaussianNormalizer(mean(𝐱, dims=dims), StatsBase.std(𝐱, dims=dims), ϵ)
+end
+
+struct Encode end
+struct Decode end
+
+(n::UnitGaussianNormalizer)(𝐱, ::Type{Encode}) = @. (𝐱-n.mean) / (n.std+n.ϵ)
+(n::UnitGaussianNormalizer)(𝐱, ::Type{Decode}) = @. 𝐱 * (n.std+n.ϵ) + n.mean
```

Comment:
```suggestion
encode(n::UnitGaussianNormalizer, 𝐱::AbstractArray) = @. (𝐱-n.mean) / (n.std+n.ϵ)
decode(n::UnitGaussianNormalizer, 𝐱::AbstractArray) = @. 𝐱 * (n.std+n.ϵ) + n.mean
```
It's better to keep the API design simple.